sdk/digitaltwins/Azure.DigitalTwins.Core/src/Models/BasicDigitalTwin.cs

```diff
@@ -86,7 +88,26 @@ public class BasicDigitalTwin
         /// A string representing a weak ETag for the entity that this request performs an operation against, as per RFC7232.
         /// </summary>
         [JsonPropertyName(DigitalTwinsJsonPropertyNames.DigitalTwinETag)]
-        public string ETag { get; set; }
+        [JsonConverter(typeof(OptionalETagConverter))] // TODO: Remove when #16272 is fixed
```

Comment:
this is interesting

Reply:
(And let's reference #16272 here so it shows up on the issue too)